uber/automated_emails.py

```diff
@@ -722,7 +722,6 @@ def __init__(self, subject, template, filter, ident, **kwargs):
         ident='2018_email_blast')
 
     MIVSGuestEmailFixture(
-        IndieGame,
```

Comment:
This was breaking the build -- MIVSEmailGuestFixture doesn't need the model defined, so it was taking the fourth parameter as the `ident`, and then in the fifth parameter we passed it the `ident`. Whoops.